packages/finicity-token-exchange/src/theme.ts

```diff
@@ -0,0 +1,16 @@
+import { createTheme } from "@mui/material";
+import { red } from "@mui/material/colors";
+
+export default createTheme({
+    palette: {
+        primary: {
+            main: "#556CD6"
```

Comment:
What if we added the Dwolla Primary color for the timebeing? 
```suggestion
            main: "#2D2D48"
```